corehq/apps/enterprise/templates/enterprise/enterprise_dashboard.html

```diff
@@ -2,7 +2,7 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
-{% block page_title %}{{ account.name }}{% endblock %}
+{% block page_title %}{{ header_title }}{% endblock %}
 
 {% requirejs_main_b5 'enterprise/js/enterprise_dashboard' %}
```

Comment:
please migrate to webpack thank you!

Reply:
Given work like https://github.com/dimagi/commcare-hq/pull/35409/, this is going to cause conflicts. I think we should handle this as a separate PR